packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -68,13 +65,16 @@ export const Subscriptions = ({
 }: SubscriptionsProps) => {
   const {
     accessToken,
+    config,
     locationReload,
     navigateToUrl,
+    queryParams
   } = useContext(AppContext);
 
   const [showPaymentSuccessAlert, setShowPaymentSuccessAlert] = useState(true);
   const clearSuccessAlert = useCallback(() => setShowPaymentSuccessAlert(false),
                                               [setShowPaymentSuccessAlert]);
+  const SUPPORT_FORM_URL = `${config.servers.content.url}/support`;
```

Comment:
@meandavejustice This should now take you to the form.  (An active subscription is still required.)